chef/cookbooks/dhcp/recipes/default.rb

```diff
@@ -60,21 +75,23 @@
 end
 
 # This needs to be evaled.
-intfs = [Chef::Recipe::Barclamp::Inventory.get_network_by_type(node, "admin").interface]
-address = Chef::Recipe::Barclamp::Inventory.get_network_by_type(node, "admin").address
+admin_network = Chef::Recipe::Barclamp::Inventory.get_network_by_type(node, "admin")
+intfs = [admin_network.interface]
```

Comment:
it is unused.. but the comment above say. "# this needs to be evaled." so maybe it's where to make sure an address exists? All I know if hound complained because it was an unused variable. Will add it back as it was there before.